worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -453,6 +453,14 @@ private[celeborn] class Worker(
       0
     }
   }
+  // Unreleased partition location count when worker is restarting
+  workerSource.addGauge(WorkerSource.UNRELEASED_PARTITION_LOCATION_COUNT) { () =>
+    if (shutdown.get()) {
+      partitionLocationInfo.primaryPartitionLocations.values()
```

Comment:
should be `primaryPartitionLocations.values().size() + replicaPartitionLocations.values().size()`

Reply:
addressed.